### ISSUE TITLE
Support for annotations?

### ISSUE THREAD

**Issue by mikeshultz** [orionvm/potsdb#8]
Could support for [annotations](http://opentsdb.net/docs/build/html/api_http/annotation/index.html) be added?

**Comment by mcclymont** [comment]
Sorry but annotations are not supported by the Telnet API which potsdb uses, and we are unlikely to switch API unless Telnet becomes deprecated.

**Comment by mikeshultz** [comment]
Fair enough.  Thanks for the response.  Guess I'll look into their HTTP API.
